references/classification/presets.py

```diff
@@ -30,9 +38,13 @@ def __init__(
             else:
                 aa_policy = autoaugment.AutoAugmentPolicy(auto_augment_policy)
                 trans.append(autoaugment.AutoAugment(policy=aa_policy, interpolation=interpolation))
+
+        if backend == "pil":
+            # Note: we could also just use pure tensors?
```

Comment:
Not sure I understand the comment? Is this a foreshadowing for datapoints? Because so far we only deal with PIL images and plain tensors.

Reply:
sorry this is copy pasted, will remove